src/test/osd/osd-reactivate.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+#
+# Author: Vicente Cheng <freeze.bilsted@gmail.com>
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Library Public License as published by
+# the Free Software Foundation; either version 2, or (at your option)
+# any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Library Public License for more details.
+#
+
+source ../qa/workunits/ceph-helpers.sh
+
+function run() {
+    local dir=$1
+    shift
+
+    export CEPH_MON="127.0.0.1:7106"
```

Comment:
@dachary 
that's a good reminder for someone want to write a new test function!!

btw your comment is `# git grep '\<7109\> : there must be only one`
maybe lost another `'`?
like that  `# git grep '\<7109\>' : there must be only one`

anyway, I think that reminder is useful!!